student.rb

```diff
@@ -0,0 +1,31 @@
+require './person'
+
```

Comment:
- [OPTIONAL] It is better to use `require_relative` instead of `require` to load files in a repository. The general rule of thumb is that `require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory. I would advise you to make this change. 👍 💻